integ-test/src/test/resources/datetime.json

```diff
@@ -3,6 +3,6 @@
 {"index":{"_id":"2"}}
 {"login_time":"2015-01-01T12:10:30Z"}
 {"index":{"_id":"3"}}
-{"login_time":"1585882955"}
+{"login_time":"1585882955000"}
```

Comment:
I found while writing the reproducers that this was getting parsed as milliseconds instead of seconds, despite the magnitude. That gets sent to January 19, 1970 instead of (probably intended) April 3, 2020. I decided to update the row.